input_parser.star

```diff
@@ -134,12 +137,22 @@ DEFAULT_L1_ARGS = {
     # "minimal" preset will spin up a network with minimal preset. This is useful for rapid testing and development.
     # 192 seconds to get to finalized epoch vs 1536 seconds with mainnet defaults
     # Please note that minimal preset requires alternative client images.
-    "l1_preset": "minimal",
+    "l1_preset": "mainnet",
     # Number of seconds per slot on the Beacon chain
     # Default: 12
     "l1_seconds_per_slot": 1,
-    # The amount of ETH sent to the admin, sequence, aggregator and sequencer addresses.
-    "l1_funding_amount": "100ether",
+    # The amount of ETH sent to the admin, sequence, aggregator, sequencer and other chosen addresses.
+    "l1_funding_amount": "1000000ether",
+    # Default: 2
+    "l1_participants_count": 1,
+    # Default: 2048
+    "l1_eth1_follow_distance": 1,
+    # Default: 256
+    "l1_min_validator_withdrawability_delay": 1,
```

Comment:
[MIN_VALIDATOR_WITHDRAWABILITY_DELAY
](https://eth2book.info/capella/part3/config/configuration/#min_validator_withdrawability_delay)A validator can stop participating once it has made it through the exit queue. However, its stake remains locked for the duration of MIN_VALIDATOR_WITHDRAWABILITY_DELAY. This is to allow some time for any slashable behaviour to be detected and reported so that the validator can still be penalised (in which case the validator's withdrawable time is pushed [EPOCHS_PER_SLASHINGS_VECTOR](https://eth2book.info/capella/part3/config/preset/#epochs_per_slashings_vector) into the future).

Once the MIN_VALIDATOR_WITHDRAWABILITY_DELAY period has passed, the validator becomes eligible for a full withdrawal of its stake and rewards on the next withdrawals sweep, as long as it has [ETH1_ADDRESS_WITHDRAWAL_PREFIX](https://eth2book.info/capella/part3/config/constants/#eth1_address_withdrawal_prefix) (0x01) withdrawal credentials set. In any case, being in a "withdrawable" state means that a validator has now fully exited from the protocol.

Reply:
Same here